build.sh

```diff
@@ -5,7 +5,7 @@ local_bin="$HOME/.local/bin/"
 local_include="$HOME/.local/include/csq"
 csq_bin="$local_bin/csq"
 csq_path="$local_include"
-shell_config="$HOME/.${SHELL##*/}rc"
```

Comment:
This approach makes your script more flexible and ensures it works with the user's current shell, regardless of which shell it is.

Reply:
At least check the `.bashrc` first before using shell based config files.
This is because most modern distro's ship with `.bashrc` as somehow a safe config file. Most of projects use `.bashrc` as a config file.

I personally use `fish` shell and some of the config are in `.bashrc` and the others are in `.config/fish`.

Reply:
Hmm.. OK add this condition is .bashrc available than use it  else use current shell_rc